apps/happy-blocks/block-library/pricing-plans/index.php

```diff
@@ -81,6 +81,45 @@ function happyblocks_pricing_plans_render_callback( $attributes ) {
 HTML;
 }
 
+/**
+ * Return the correct asset relative path to determine the translation file name,
+ * when loading the translation files from wp.com CDN.
+ *
+ * @param string|false $relative The relative path of the script. False if it could not be determined.
+ * @param string       $src      The full source URL of the script.
+ * @return string|false          The new relative path
+ */
+function happyblocks_pricing_plans_normalize_translations_relative_path( $relative, $src ) {
+	// Rewrite our CDN path to a relative path to calculate the right filename.
+	if ( preg_match( '#https?://s[0123]\.wp\.com/wp-content/a8c-plugins/happy-blocks/(block-library.*\.js)#', $src, $m ) ) {
+		return $m[1];
```

Comment:
I noticed that when the plugin is synced to a sandbox in dev environment (i.e. `yarn dev --sync`), block assets are placed in an additional sub-directory `build`, which causes the relative path to no longer match the generated language files.

Do you think we can do something like `str_replace( 'build/', '', $m[1] )` here to cover that case?

Reply:
Sure, I've added it as well.